misc/test_site/mac_addrs/3c5ab41e8f0a/module_config.json

```diff
@@ -18,7 +18,10 @@
       "enabled": true
     },
     "switch": {
-      "enabled": true
+      "enabled": true,
+      "poe": {
```

Comment:
I don't think this makes sense for the default integration tests (aka test_site)?  It can't do PoE... ?

Reply:
Removed poe entry in file.

```
"switch": {
      "enabled": true
    }
```